src/components/skDashboard/styles/SocialMediaCard.styled.ts

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components'
+
+export const StyledCard = styled.div`
+  color: ${({ theme }) => theme.headingText};
+  border-top: 6px solid;
+  border-top-color: hsl(208, 92%, 53%);
+  line-height: 20px;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border-radius: 10px;
+  filter: drop-shadow(0px 0px 4px rgba(0, 0, 0, 0.15));
+  cursor: pointer;
+  transition: all 0.3s cubic-bezier(0, 0, 0.5, 1);
+  justify-content: center;
+  align-items: center;
+  margin: 2.5px;
+  width: 18vw;
+  height: 30vh;
+  margin-top: 2vh;
+  margin-bottom: 2vh;
```

Comment:
why are there 3 lines of margin, is it suppose to be 2vh or 2.5px? consider doing all in 1 line instead

Reply:
a95d557